components/inputs/test/input-select.visual-diff.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../typography/typography.js';
+		</script>
+		<title>d2l-input-select visual-diff tests</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<style>
+			select.vui-input {
+    border-radius: .3rem;
+    border-style: solid;
+    box-sizing: border-box;
+    display: inline-block;
+    height: auto;
+    margin: 0;
+    vertical-align: middle;
+    transition: background-color .5s ease,border-color 1ms ease;
+    color: #494c4e;
+    font-family: inherit;
+    font-size: .8rem;
+    font-weight: 400;
+    letter-spacing: .02rem;
+    line-height: 1.2rem
+}
+select.vui-input::-webkit-input-placeholder {
+    color: #cdd5dc
+}
+select.vui-input::-moz-placeholder {
+    color: #cdd5dc
+}
+select.vui-input:-ms-input-placeholder {
+    color: #cdd5dc
+}
+select.vui-input::-ms-input-placeholder {
+    color: #cdd5dc
+}
+select.vui-input::placeholder {
+    color: #cdd5dc
+}
+select.vui-input,select.vui-input:hover:disabled {
+    background-color: #fff;
+    border-color: #868c8f;
+    border-width: 1px;
+    box-shadow: inset 0 2px 0 0 rgba(181,189,194,.2);
+    padding: .4rem .75rem
+}
+select.vui-input.vui-input-focus,select.vui-input:focus,select.vui-input:hover {
+    border-color: #006fbf;
+    border-width: 2px;
+    outline-width: 0;
+    padding: calc(.4rem - 1px) calc(.75rem - 1px)
+}
+select.vui-input[aria-invalid=true] {
+    border-color: #cd2026
+}
+select.vui-input:disabled {
+    opacity: .5
+}
+select.vui-input::-webkit-search-cancel-button {
+    display: none
+}
+select.vui-input::-ms-clear {
+    display: none;
+    width: 0;
+    height: 0
+}
+select.vui-input option {
+    font-weight: 400
+}
+@media screen and (-webkit-min-device-pixel-ratio: 0),screen and (min--moz-device-pixel-ratio:0),screen and (-ms-high-contrast:active),(-ms-high-contrast:none) {
```

Comment:
I'm debating not bringing this media query forward, as I'm actually not sure what it's used for. It _might_ be trying to exclude older versions of IE, like IE10? If so... 🔥

Reply:
And by "not bring it forward" I mean roll all this CSS into the rules above, without the media query.

Reply:
Found [some useful comments](https://github.com/Brightspace/valence-ui-input/commit/f0d02b2e5634ee8374318edc90db04990c7d1d61#diff-e7d65af6d7a161dd41f69c2306f67de2) in an ancient version of this. Looks like it was indeed so that older IE wouldn't explode, and we used to have separate CSS for Chrome/Safari and Firefox. I'm going to try and kill it.